pkg/util/util_test.go

```diff
@@ -255,6 +256,10 @@ func testCompareEndpointSpec() {
 
 func testGetDefaultGatewayInterface() {
 	It("should find and return a non-nil Interface object", func() {
-		Expect(util.GetDefaultGatewayInterface()).NotTo(BeNil())
+		gatewayInterface, err := util.GetDefaultGatewayInterface()
+		if err == netlink.ErrNotImplemented {
+			Skip("netlink module does not implement RouteList on this OS")
+		}
+		Expect(gatewayInterface).NotTo(BeNil())
```

Comment:
I'd actually prefer just to get rid of this test case as it really isn't all that useful since all we can do is verify it returns non-nil. In fact I omitted this in the Testify version but neglected to remove it in the Gingko version prior to merging.

Reply:
Ack, I will remove it in this patch.